Flow.Launcher.Core/Plugin/PortableSettings.cs

```diff
@@ -0,0 +1,415 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Documents;
+using Flow.Launcher.Infrastructure.Storage;
+using Flow.Launcher.Plugin;
+
+namespace Flow.Launcher.Core.Plugin
+{
+    public class PortableSettings
```

Comment:
Ideally because I don't feel like this may be only used by jsonrpc plugin. Portable means it is applicable to any kinds of plugin but sure if you want to change the name. I have a hard time to figure out names.

Reply:
Hmm portable to me means this settings can be moved anyway, but in actuality this class is used for  all non-native plugins right? So all JsonRPC plugins will use it, I don't suppose native plugins will want to set up their settings window this way? Or am I missing a part of the equation?

Reply:
Non-native plugin doesn't have to be JsonRPC. Though that's probably the case unless we have some special plugins.